keys.go

```diff
@@ -102,25 +113,21 @@ func (c *Client) CreateKey(ctx context.Context, name string, expiration *time.Ti
 	return c.CreateImportedKey(ctx, name, expiration, "", "", "", extractable)
 }
 
+// CreateKeyWithSHA1 creates a new KP key using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateKeyWithSHA1(ctx context.Context, name string, expiration *time.Time, extractable bool) (*Key, error) {
+	return c.CreateImportedKeyWithSHA1(ctx, name, expiration, "", "", "", extractable)
+}
+
 // CreateImportedKey creates a new KP key from the given key material.
 func (c *Client) CreateImportedKey(ctx context.Context, name string, expiration *time.Time, payload, encryptedNonce, iv string, extractable bool) (*Key, error) {
-	key := Key{
-		Name:        name,
-		Type:        keyType,
-		Extractable: extractable,
-		Payload:     payload,
-	}
-
-	if payload != "" && encryptedNonce != "" && iv != "" {
-		key.EncryptedNonce = encryptedNonce
-		key.IV = iv
-		key.EncryptionAlgorithm = importTokenEncAlgo
-	}
-
-	if expiration != nil {
-		key.Expiration = expiration
-	}
+	key := c.createKeyTemplate(ctx, name, expiration, payload, encryptedNonce, iv, extractable, nil, AlgorithmRSAOAEP256)
+	return c.createKey(ctx, key)
+}
 
+// CreateImportedKeyWithSHA1 creates a new KP key from the given key material
+// using RSAES OAEP SHA 1 as encryption algorithm.
+func (c *Client) CreateImportedKeyWithSHA1(ctx context.Context, name string, expiration *time.Time, payload, encryptedNonce, iv string, extractable bool) (*Key, error) {
```

Comment:
Lets just use this method, and add the aliases as a parameter, which lets us drop the `WithAliases` function below.

We would actually would have liked to do this with the normal create key, but that function is used already by many, so we can't easily change that function signature. This function is new so we can add the parameter easily enough.